### PR TITLE
docs(samples): add initial sample codes

### LINKED ISSUE #12: Create sample codes
Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

 **Is your feature request related to a problem? Please describe.**
Create sample codes for this library.
 **Describe the solution you'd like**
N/A
 **Describe alternatives you've considered**
N/A
 **Additional context**
N/A

### PR DESCRIPTION
**Pull request by wuyuexin**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-dialogflow-cx/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #12  🦕

### CODE REVIEW

File: samples/snippets/detect_intent_stream.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import argparse
+import uuid
+
+from google.cloud.dialogflowcx_v3beta1.services.sessions import SessionsClient
+from google.cloud.dialogflowcx_v3beta1.types import audio_config
+from google.cloud.dialogflowcx_v3beta1.types import session
+
+
+# [START dialogflow_detect_intent_stream]
+def detect_intent_stream(agent, session_id, audio_file_path, language_code):
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_stream.py:34]: Please add some hints to the user to what these values mean. You can do one of either of the following:

(preferred) Add a simple overloaded method that demonstrates initalization and calles the underlying function:
```python 
# [START dialogflow_detect_intent_stream]
def run_sample():
    # TODO(developer): Replace these values when running the function
    agent = "YOUR-AGENT-ID"  # https://link.to.more.info.on.agents
    # ...
    detect_intent_stream(agent, session_id, audio_file_path, language_code):

def detect_intent_stream(agent, session_id, audio_file_path, language_code):
```

File: samples/snippets/detect_intent_stream.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    """Returns the result of detect intent with streaming audio as input.
+
+    Using the same `session_id` between requests allows continuation
+    of the conversation."""
+    session_client = SessionsClient()
+    session_path = "{}/sessions/{}".format(agent, session_id)
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_stream.py:40]: Please use f-strings in modern python versions:
```suggestion
    session_path = f"{agent}/sessions/{session_id}"
```

File: samples/snippets/detect_intent_stream.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                chunk = audio_file.read(4096)
+                if not chunk:
+                    break
+                # The later requests contains audio data.
+                audio_input = session.AudioInput(audio=chunk)
+                query_input = session.QueryInput(audio=audio_input)
+                yield session.StreamingDetectIntentRequest(query_input=query_input)
+
+    requests = request_generator()
+    responses = session_client.streaming_detect_intent(requests=requests)
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_stream.py:71]: nit: inline this argument
```suggestion
    responses = session_client.streaming_detect_intent(requests=request_generator())
```

File: samples/snippets/detect_intent_stream.py (hunk 2 above)
Comment (kurtisvg) [reply samples/snippets/detect_intent_stream.py:40]: (same for other places)

File: samples/snippets/detect_intent_stream.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    print(
+        "Response text: {}\n".format(
+            " ".join(
+                [
+                    " ".join(response_message.text.text)
+                    for response_message in response.query_result.response_messages
+                ]
+            )
+        )
+    )
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_stream.py:94]: nit: This is way more readable if you break into different lines
```suggestion
    response_messages = [" ".join(msg.text.text) for msg in response.query_result.response_messages]
    print(f"Response text: {" ".join(response_messages}\n")
```

File: samples/snippets/detect_intent_stream_test.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+
+from detect_intent_stream import detect_intent_stream
+
+DIRNAME = os.path.realpath(os.path.dirname(__file__))
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+AGENT_ID = "53516802-3e2a-4016-80b6-a3df0d240240"
+AGENT = "projects/{}/locations/global/agents/{}".format(PROJECT_ID, AGENT_ID)
+SESSION_ID = uuid.uuid4()
+AUDIO = "{}/resources/hello.wav".format(DIRNAME)
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_stream_test.py:29]: The location for external resources for the test should be passed in by env vars - See [External Resources](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#external-resources)

**Review by kurtisvg — CHANGES_REQUESTED** [review]

File: samples/snippets/detect_intent_audio.py [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+from google.cloud.dialogflowcx_v3beta1.types import session
+
+
+def run_sample():
+    # TODO(developer): Replace these values when running the function
+    project_id = "YOUR-PROJECT-ID"
+    location_id = "YOUR-LOCATION-ID"
+    agent_id = (
+        "YOUR-AGENT-ID"  # https://cloud.google.com/dialogflow/cx/docs/concept/agent
+    )
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_audio.py:39]: nit:
```suggestion
    # For more info on agents see https://cloud.google.com/dialogflow/cx/docs/concept/agent
    agent_id = "YOUR-AGENT-ID" 
```

File: samples/snippets/detect_intent_audio.py [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+    agent = f"projects/{project_id}/locations/{location_id}/agents/{agent_id}"
+    session_id = str(
+        uuid.uuid4()
+    )  # https://cloud.google.com/dialogflow/cx/docs/concept/session
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_audio.py:43]:
```suggestion
    # For more information on sessions see https://cloud.google.com/dialogflow/cx/docs/concept/session
    session_id = str(uuid.uuid4())  
```

File: samples/snippets/detect_intent_audio.py [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+    audio_file_path = "YOUR-AUDIO-FILE-PATH"
+    language_code = "en-us"
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_audio.py:45]: Is this always `en-us`? If it is, can we move below into the sample? If not, can add a link to a list of supported options?

File: samples/snippets/detect_intent_audio.py [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+    detect_intent_audio(agent, session_id, audio_file_path, language_code)
+
+
+# [START dialogflow_detect_intent_audio]
```
Comment (kurtisvg) [inline samples/snippets/detect_intent_audio.py:50]: This region tag should be above the `run_sample` function

**Review by kurtisvg — CHANGES_REQUESTED** [review]

**Comment by kurtisvg** [comment]
@busunkim96 I don't see any Kokoro tests running on this repo. Is that intentional?

**Comment by busunkim96** [comment]
@kurtisvg Kokoro isn't running is because author is outside the org. I also need to add samples configs though. Opened 334490453.

**Comment by kurtisvg** [comment]
> @kurtisvg Kokoro isn't running is because author is outside the org. I also need to add samples configs though. Opened 334490453.

That was specifically what I was referring too :)

**Review by kurtisvg — APPROVED** [review]

**Comment by kurtisvg** [comment]
@wuyuexin Let me know if you need help merging this.

**Comment by wuyuexin** [comment]
> @wuyuexin Let me know if you need help merging this.

Yes, I'm not familiar with the merging process. Is there anything I need to do?

**Comment by busunkim96** [comment]
👋  There's a small tweak I want to make to the `noxfile`/`noxfile_config`. I'll merge it for you right after that @wuyuexin.

Side note, I think I'll have to add another PR template for samples that looks like the one in python-docs-samples.

**Review by busunkim96 — APPROVED** [review]

**Pull request merged**